src/db.cpp

```diff
@@ -65,24 +66,23 @@ bool CDBEnv::Open(boost::filesystem::path pathEnv_)
         return true;
 
     boost::this_thread::interruption_point();
-
     pathEnv = pathEnv_;
     filesystem::path pathDataDir = pathEnv;
     strPath = pathDataDir.string();
     filesystem::path pathLogDir = pathDataDir / "database";
     filesystem::create_directory(pathLogDir);
     filesystem::path pathErrorFile = pathDataDir / "db.log";
-    LogPrintf("dbenv.open LogDir=%s ErrorFile=%s\n", pathLogDir.string(), pathErrorFile.string());
+    LogPrintf("CDBEnv::Open : LogDir=%s ErrorFile=%s\n", pathLogDir.string(), pathErrorFile.string());
 
     unsigned int nEnvFlags = 0;
     if (GetBoolArg("-privdb", true))
         nEnvFlags |= DB_PRIVATE;
 
     int nDbCache = GetArg("-dbcache", 25);
```

Comment:
Check commit https://github.com/TheLindaProjectInc/Linda/commit/99c5d133d0b52e7788fa1bd3b1e98029a6a41863
This removed the nDBCache variable as it was unused.
It appears that this will now be utilized(line 85), so will need adding back in.